test/md_test.rb

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env rspec
+# encoding: utf-8
+
+# Copyright (c) [2017] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require_relative "spec_helper"
+require "y2storage"
+require "y2storage/md_level"
+require "y2storage/md_parity"
+
+describe Y2Storage::Md do
+  using Y2Storage::Refinements::SizeCasts
+
+  before do
+    fake_scenario(scenario)
+  end
+
+  let(:scenario) { "md2-devicegraph.xml" }
+  let(:md_name) { "/dev/md0" }
+  subject(:md) { Y2Storage::Md.find_by_name(fake_devicegraph, md_name) }
+
+  describe "#devices" do
+
+    it "returns the array of BlkDevices use" do
+      # TODO: check the complete return value
+      expect(md.devices.size).to eq 2
+    end
+
+  end
+
+  describe "#numeric?" do
+
+    it "returns true for /dev/md0" do
+      expect(md.numeric?).to eq true
+    end
+
+  end
+
+  describe "#get_number" do
```

Comment:
this should be `describe "#number" do` as it describe ruby API and not libstorage one.